processor-steps/src/main/java/boringyuri/processor/common/steps/UriFactoryGeneratorStep.kt

```diff
@@ -161,34 +173,36 @@ class UriFactoryGeneratorStep internal constructor(
     }
 
     private fun createParamSpecs(
-        methodParameters: List<VariableElement>
-    ): Map<VariableElement, ParameterSpec> {
+        methodParameters: List<XVariableElement>
+    ): Map<XVariableElement, ParameterSpec> {
         return methodParameters.associateWithTo(LinkedHashMap()) { parameter ->
             parameter.createParamSpec(annotationHandler)
         }
     }
 
     private fun obtainPathSegments(
-        methodParameters: List<VariableElement>,
-        parameterSpecs: Map<VariableElement, ParameterSpec>
+        methodParameters: List<XVariableElement>,
+        parameterSpecs: Map<XVariableElement, ParameterSpec>
     ): Map<String, VariableWritePathSegment> {
         return methodParameters.mapNotNull { param ->
             val pathAnnotation = param.getAnnotation<Path>() ?: return@mapNotNull null
 
             val spec = parameterSpecs.getValue(param)
             val nullable = annotationHandler.isNullable(spec.type, param)
-            val defaultValue = param.getAnnotation<DefaultValue>()?.value
+            val defaultValue = param.getAnnotation<DefaultValue>()
```

Comment:
nit: you may keep this line as it was
```suggestion
            val defaultValue = param.getAnnotation<DefaultValue>()?.value
```

Reply:
done